modules/related-posts/jetpack-related-posts.php

```diff
@@ -1279,57 +1240,52 @@ protected function _enqueue_assets( $script, $style ) {
 			);
 			wp_localize_script( 'jetpack_related-posts', 'related_posts_js_options', $related_posts_js_options );
 		}
-		if ( $style ){
-			if( is_rtl() ) {
+		if ( $style ) {
+			if ( is_rtl() ) {
 				wp_enqueue_style( 'jetpack_related-posts', plugins_url( 'rtl/related-posts-rtl.css', __FILE__ ), array(), self::VERSION );
 			} else {
 				wp_enqueue_style( 'jetpack_related-posts', plugins_url( 'related-posts.css', __FILE__ ), array(), self::VERSION );
 			}
 		}
 	}
-
 	/**
 	 * Sets up the shortcode processing.
 	 *
 	 * @uses add_filter, add_shortcode
-	 * @return null
+	 * @return void
 	 */
 	protected function _setup_shortcode() {
 		add_filter( 'the_content', array( $this, 'test_for_shortcode' ), 0 );
-
 		add_shortcode( self::SHORTCODE, array( $this, 'get_target_html' ) );
 	}
 
+	/**
```

Comment:
This is wrong. This PHPdoc block is for the filter, not for the method. Don't move it!
You can write the PHPdoc for the function yourself.